purchase_packaging/models/purchase_order_line.py

```diff
@@ -53,33 +53,44 @@ def _compute_product_qty(self):
         """
         Compute the total quantity
         """
+        uom_categories = self.mapped("product_purchase_uom_id.category_id")
```

Comment:
@tfossoulw I think this is good but you should put product_purchase_uom_id.category_id in the depends as recordset would be already filled in with these records before entering function.

Reply:
(As your 'mapped' could induce a query if records are not fetched)

Reply:
@rousseldenis done :)